core/services/ocr2/plugins/directrequestocr/test/internal/testutils.go

```diff
@@ -99,6 +103,52 @@ func SetConfig(t *testing.T, owner *bind.TransactOpts, oracleContract *ocr2dr_or
 	require.NoError(t, err)
 }
 
+func SetRegistryConfig(t *testing.T, owner *bind.TransactOpts, registryContract *ocr2dr_registry.OCR2DRRegistry, oracleContract *ocr2dr_oracle.OCR2DROracle) {
+	var maxGasLimit =  uint32(1_000_000)
+	var stalenessSeconds =  uint32(86_400)
+	var gasAfterPaymentCalculation = big.NewInt(21_000 + 5_000 + 2_100 + 20_000 + 2 * 2_100 - 15_000 + 7_315)
+	var weiPerUnitLink =  big.NewInt(5000000000000000)
+	var gasOverhead =  uint32(500_000)
+
+	_, err := registryContract.SetConfig(
+		owner,
+		maxGasLimit,
+		stalenessSeconds,
+		gasAfterPaymentCalculation,
+		weiPerUnitLink,
+		gasOverhead,
+	)
+	require.NoError(t, err)
+
+	var senders []common.Address
+	senders[0] = oracleContract.Address()
```

Comment:
Unfortunately, oracleContract.Address() doesn't give you what you want... Just pass oracelContractAddress from the test.